AWSTranscribeStreaming/AWSTranscribeStreamingService.h

```diff
@@ -186,9 +186,9 @@ For example, set the default service configuration in `- application:didFinishLa
                                                                                                         identityPoolId:@"YourIdentityPoolId"];
         AWSServiceConfiguration *configuration = [[AWSServiceConfiguration alloc] initWithRegion:AWSRegionUSWest2
                                                                              credentialsProvider:credentialsProvider];
-        WebSocketProvider = provider [[WebSocketProvider alloc] init;
+        NativeWebSocketProvider  webSocketProvider =  [[NativeWebSocketProvider alloc] init;
```

Comment:
nit: change `webSocketProvider` to `*webSocketProvider` & remove extra space.

Reply:
fixed